transport/rif_comms/client.py

```diff
@@ -77,11 +70,10 @@ def send_message(self, payload: str, rsk_address: Address):
         :param payload: the message data to be sent
         :param rsk_address: the destination for the message to be sent to
         """
-        topic_id = self._get_peer_id(to_checksum_address(rsk_address))
         # TODO: message encoding
```

Comment:
i know its not related with this pr but i think we can remove this TODO

Reply:
agreed

Reply:
done